README.md

```diff
@@ -1,8 +1,8 @@
 # ***imgman***
 
-imgman is an image manipulation website built with django, which provides options for converting an image to blur, black and white, grayscale, pdf.
+imgman is a unique web app which is provides myriad of image manipulation options like blurring of image, black and white filters, converting to pdf, adding grayscale eefect. And it's built with django a python framework.
```

Comment:
```suggestion
imgman is a unique web app which provides myriad of image manipulation options like blurring of image, black and white filters, converting to pdf, adding grayscale effect, resizing the image. It's built with django, a python web framework.
```

Reply:
Please Resolve these errors @tamoghna-dey

Reply:
Yes sure, do I need to re-edit in the same PR or make another and edit it again?

Reply:
> Yes sure, do I need to re-edit in the same PR or make another and edit it again?

yes you can also click on the commit suggestion button